mlflow/tracking/registry.py

```diff
@@ -74,7 +74,9 @@ def get_store_builder(self, store_uri):
             URI requirements.
         """
         scheme = (
-            store_uri if store_uri in {"databricks", "databricks-uc"} else get_uri_scheme(store_uri)
+            store_uri
+            if store_uri in {"databricks", "databricks-uc", "uc"}
```

Comment:
Curious if it would be less ambiguous to name it `oss-uc` as compared to `databricks-uc`?

Reply:
We're relying on a lot of the existing handling of the `databricks-uc` scheme checks once we make to this point.  I think its ok because the only time you would use this nomenclature is if you are in the databricks environment.